shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -888,16 +895,31 @@ - (instancetype)initWithTarget:(id)target
 }
 
 - (void)touchesBegan:(NSSet*)touches withEvent:(UIEvent*)event {
-  [_platformViewsController->getFlutterViewController() touchesBegan:touches withEvent:event];
+  if (!_flutterViewController) {
```

Comment:
Wouldn't it be safer to always assign _flutterViewController in `touchesBegan` and nil it out in cancelled and ended?  It seems like this way we could accidentally forward touches to the wrong flutterviewcontroller.